model_analyzer/reports/report_manager.py

```diff
@@ -74,103 +71,120 @@ def export_summary(self,
-
-            summary.add_paragraph(
-                "The following table summarizes each configuration at the measurement"
-                " that optimizes the desired metrics under the given constraints in"
-                " decreasing order of throughput.")
-            summary.add_table(table=self._build_summary_table(
-                model_name=model_name))
+                                               'throughput_v_latency.png')
+        caption1 = f"Throughput vs. Latency curves for {num_best_configs} configurations of {model_name}"
+        memory_latency_plot = os.path.join(self._config.export_path, 'plots',
+                                           model_name, 'gpu_mem_v_latency.png')
+        caption2 = f"GPU Memory vs. Latency curves for {num_best_configs} configurations of {model_name}"
+        summary.add_images([throughput_latency_plot, memory_latency_plot],
+                           [caption1, caption2])
+        summary.add_paragraph(
+            "The maximum GPU memory consumption for each of the above points is"
+            f" shown in the second plot. The GPUs {gpu_names} have"
+            f" a total available memory of {max_memories} respectively.")
+
+        summary.add_paragraph(
+            "The following table summarizes each configuration at the measurement"
+            " that optimizes the desired metrics under the given constraints in"
+            " decreasing order of throughput.")
+        summary.add_table(table=table)
         return summary
 
-    def _build_summary_table(self, model_name):
+    def _build_summary_table(self, model_name, num_measurements, gpu_name):
         """
         Creates a result table corresponding
         to the best measurements for a particular
         model
         """
 
         summary_table = ResultTable(headers=[
-            'Model Config Name', 'Dynamic Batcher Sizes', 'Instance Count',
+            'Model Config Name', 'Max Dynamic Batch Size', 'Instance Count',
             'p99 Latency (ms)', 'Throughput (infer/sec)',
-            'GPU Memory Usage (MB)', 'GPU Utilization (%)'
+            'Max GPU Memory Usage (MB)', 'Average GPU Utilization (%)'
         ],
                                     title="Report Table")
 
+        best = True
+        summary_sentence = ""
         for model_config, measurement in self._measurements[model_name]:
             dynamic_batching_str = model_config.dynamic_batching_string()
             instance_group_str = model_config.instance_group_string()
+            if best:
+                summary_sentence = (
+                    f"In {num_measurements} measurements "
+                    f"{instance_group_str} model instances at concurrency "
+                    f"{measurement.perf_config()['concurrency-range']} "
+                    f"and max dynamic batch size of {dynamic_batching_str} "
+                    f"on platform {model_config.get_field('platform')} delivers "
```

Comment:
platform is going to be deprecated in Triton soon. I think only TF-TRT is using the platform field. Other backends are using the `backend` field. You can check here whether the `backend` field exist if it doesn't exist you can use `platform`.